extensions/sql-database-projects/src/controllers/projectController.ts

```diff
@@ -540,9 +540,11 @@ export class ProjectsController {
 			return undefined; // buildProject() handles displaying the error
 		}
 
-		// copy dacpac to temp location before publishing
-		const tempPath = path.join(os.tmpdir(), `${path.parse(dacpacPath).name}_${new Date().getTime()}${constants.sqlprojExtension}`);
-		await fs.copyFile(dacpacPath, tempPath);
+		// copy entire build output to temp location before publishing
+		const tempDir = path.join(os.tmpdir(), `${path.parse(dacpacPath).name}_${new Date().getTime()}`);
+		await fs.mkdir(tempDir);
```

Comment:
What are we doing the Temp directory after the publishing done? Are we cleaning it here or in DacFx? Thanks